### PR TITLE
feat(board-item): add tooltip case in overlay icon button

### PR DESCRIPTION
**Pull request by piggggggggy**
### To Reviewers
- [ ] Skipping review is not a problem (```style```, ```chore``` ONLY)
- [ ] Not that difficult

### Type of Change
- [x] New feature
- [ ] Bug fixes
- [ ] Feature improvement
  - [ ] These changes require a usage change
- [ ] Others (performance improvement, refactoring, CI/CD, etc.)

### Checklist
- [x] Updated Storybook documents
- [x] Tested with console(if usages are changed) 

### Description
SSIA

image

### CODE REVIEW

**Review by yuda110 — APPROVED** [review]
LGTM!!!!

**Review by minseolee — APPROVED** [review]
LGTM!

File: src/data-display/board-item/type.ts [hunk 1]
```diff
@@ -8,5 +8,6 @@ export type ButtonEventHandler = (...args: any[] | any) => Promise<void> | void;
 
 export interface IconSet {
     iconName: string;
+    tooltipText?: string;
```
Comment (WANZARGEN) [inline src/data-display/board-item/type.ts]: nitpick: I think it would be safer to specify it as a `TranslationResult` type.

**Review by WANZARGEN — APPROVED** [review]
Nice clean code :)
I left a nitpick comment~

File: src/data-display/board-item/type.ts (hunk 1 above)
Comment (piggggggggy) [reply src/data-display/board-item/type.ts]: Thanks!!

**Pull request merged**